### DISCUSSION TITLE
Distance between vehicles / Abstand der Fahrzeuge

### DISCUSSION THREAD

**Discussion by Williwillswisse** [Mogli12/AIVehicleExtension#262] [Ideas]
Hi Stefan, gibt es die Möglichkeit den Abstand zwischen z.B. den Dreschern auf dem Bild zu beeinflussen?
Kann ich den in einer xml oder irgendwo im lua-Code ändern?
Wäre praktisch wenn der in den Settings eingestellt werden könnten.

![image](https://user-images.githubusercontent.com/47098121/77817794-140a1380-70ce-11ea-9634-3a83aa8ff0e8.png)

**Comment by Justice-Molton** [comment]
Würde ich auch eine coole Erweiterung finden.

**Comment by Blacksnake51** [comment]
+1 - Wäre sehr cool, wenn das umgesetzt werden könnte.

**Comment by Mogli12** [comment]
Hallo,

hier müsste man die Kollisionsbox ändern. Das ist nicht so schön.

Viele Grüße

**Comment by Williwillswisse** [comment]
Da habe ich mir schon eine verlängerte zur Seite gelegt 😉

Danke

Gruß Willi

**Comment by Mogli12** [comment]
Warum soll der Abstand denn größer werden?

**Comment by Williwillswisse** [comment]
Wie du sicher schon mitbekommen hast bin ich viel mit Autodrive unterwegs. Wenn ich dann 2, 3, 4 oder 5 Erntefahrzeuge unterwegs habe und die fahren zu dicht beieinander, dann verheddern sich die Abfahrer so leicht. Siehe Bild oben. Wenn man den Abstand einstellen könnte, könnte man zwischen den Fahrzeugen mehr Platz lassen.

**Comment by Blacksnake51** [comment]
> Warum soll der Abstand denn größer werden?

Gerade an Ecken, blockieren sich die Drescher und AD Abfahrer oftmals. Da wäre ein größerer Abstand sinnvoll. Danke
